continuous_integration/install.sh

```diff
@@ -73,6 +73,9 @@ if [ -z "$NUMPY_VERSION" ]; then
     # setup.cfg so doing it the hacky way ...
     cat setup.cfg | grep -v 'with-doctest=' > setup.cfg.new
     mv setup.cfg{.new,}
+else
```

Comment:
> why don't we always install memory_profiler ?

I think it's because at the beginning we didn't to add to much load on the CI VMs. But I'm fine with activating it in all builds.